pypowsybl/loadflow.py

```diff
@@ -348,4 +348,13 @@ def get_default_provider() -> str:
     Returns:
         the name of the current default loadflow provider
     """
-    return _pypowsybl.get_default_loadflow_provider()
\ No newline at end of file
+    return _pypowsybl.get_default_loadflow_provider()
+
+def get_provider_names() -> _List[str]:
+    """
+    Get list of supported provider names
+
+    :return: the list of supported provider names
```

Comment:
Just a detail: we should use "google style" docstrings as in the rest of the project!
Also the return type is not required since already defined in the method signature.
(see other docstrings)